File: ployglots/mta-ployglot-pkg-app-yaml-config/packages/mjs/src/logger.mjs
export function createLogger(name = 'app-yaml-config', _file) {
  const tag = name ? `[${name}]` : '';
  return {
    debug: (...args) => console.debug(tag, ...args),
    info: (...args) => console.info(tag, ...args),
    warn: (...args) => console.warn(tag, ...args),
    error: (...args) => console.error(tag, ...args),
  };
}

File: ployglots/mta-ployglot-pkg-app-yaml-config/packages/py/src/app_yaml_config/logger.py
"""Logger factory — twin of packages/mjs/src/logger.mjs."""

from __future__ import annotations

import logging
from typing import Any


def create_logger(name: str = "app-yaml-config", _file: Any = None) -> logging.Logger:
    """Return a stdlib logger for the given name. Levels: debug, info, warn(ing), error."""
    logger = logging.getLogger(name)
    if not logger.handlers:
        handler = logging.StreamHandler()
        handler.setFormatter(logging.Formatter("[%(name)s] %(message)s"))
        logger.addHandler(handler)
    return logger
